packages/web-api/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@slack/web-api",
-  "version": "7.11.0",
+  "version": "7.11.0-featWorkObjectsBeta.2",
```

```suggestion
  "version": "7.11.0",
```

🪓 todo: We ought revert this before merge!